doc/release-notes/5.9-release-notes.md

```diff
@@ -0,0 +1,157 @@
+
+## Release Highlights
+
+### Dataverse Collection Optimizations
+
+The Dataverse Collection page, which also serves as the search page and the homepage in most Dataverse installations, has been optimized, with a specific focus on reducing the number of queries for each page load.  These optimizations will be more noticable on Dataverse installations with higher traffic.
+
+### Support for HTTP "Range" Header for Partial File Downloads
+
+Dataverse now supports the HTTP "Range" header, which allows users to download parts of a file. Here are some examples:
+
+- `bytes=0-9` gets the first 10 bytes.
+- `bytes=10-19` gets 10 bytes from the middle.
+- `bytes=-10` gets the last 10 bytes.
+- `bytes=9-` gets all bytes except the first 10.
+
+Only a single range is supported. For more information, see the [Data Access API](https://guides.dataverse.org/en/5.9/api/dataaccess.html) section of the API Guide.
+
+### Support for Optional External Metadata Validation Scripts
+
+The Dataverse software now allows an installation administrator to provide custom scripts for additional metadata validation when datasets are being published and/or when Dataverse collections are being published or modified. The Harvard Dataverse Repository has been using this mechanism to combat content that violates our Terms of Use, specifically spam content. All the validation or verification logic is defined in these external scripts, thus making it possible for an installation to add checks custom-tailored to their needs.  
+
+Please note that only the metadata are subject to these validation checks. This does not check the content of any uploaded files.
+
+For more information, see the [Database Settings](https://guides.dataverse.org/en/5.9/installation/config.html) section of the Guide.
+
+### Displaying Author's Identifier as Link
+
+In the dataset page's metadata tab the author's identifier is now displayed as a clickable link, which points to the profile page in the external service (ORCID, VIAF etc.) in cases where the identifier scheme provides a resolvable landing page. If the identifier does not match the expected scheme, a link is not shown.
+
+### Auxiliary File API Enhancements
+
+This release includes updates to the Auxiliary File API. These updates include:
+
+- Auxiliary files can now also be associated with non-tabular files
+- Auxiliary files can now be deleted
+- Duplicate Auxiliary files can no longer be created
+- A new API has been added to list Auxiliary files by their origin
+- Improved error reporting
+- A bugfix involving checksums for Auxiliary files
+
+Please note that the auxiliary files feature is experimental and is designed to support integration with tools from the [OpenDP Project](https://opendp.org). If the API endpoints are not needed they can be blocked.
+
+## Major Use Cases and Infrastructure Enhancements
+
+Newly-supported major use cases in this release include:
```

Comment:
Just wondering if the first four bullets under "major use cases" here should match (and be in the same order as) the "release highlights" above. If not, it makes me wonder what order these bullets are in. Not a big deal. Just something to think about.

Reply:
reordered in https://github.com/IQSS/dataverse/pull/8284/commits/dbb7423c889bd74ee4a172a188237238f9978bcb and added a use case for the feature added by @pkiraly